src/main/java/sggw/wzim/czasnawypad/db/dto/AttractionRatingDTO.java

```diff
@@ -0,0 +1,25 @@
+package sggw.wzim.czasnawypad.db.dto;
+
+import java.time.LocalDate;
+import java.util.Date;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import lombok.*;
+
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@ToString
+@EqualsAndHashCode
+public class AttractionRatingDTO {
+    private Integer id;
+    private Integer attractionId;
+    private String attractionName;
+    private Integer rating;
+    private String notes;
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd")
+    private Date date;
```

Comment:
Don't use outdated `java.util.Date`. Please use `LocalDate` from `java.time` package

Reply:
LocalDate caused problems with jackson, then I ve added dependency but had problems with flyway I guess so I ve decided to go with this outdated java util ;/